src/UnitTests/test_quote_prov.cpp

```diff
@@ -1,25 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "unit_test.h"
 #include "dcap_provider.h"
+#include "ext\intel\sgx_ql_lib_common.h"
 #include "local_cache.h"
-#include "UnitTests/unit_test.h"
 
+#if defined(__LINUX__)
+#include <dlfcn.h>
+#include <tgmath.h>
 #include <cstdio>
 #include <cstring>
 #include <ctime>
 #include <memory>
-#include <dlfcn.h>
-#include <tgmath.h>
-
-#include "intel/sgx_ql_lib_common.h"
+#else
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <windows.h>
+#include <memory>
+#endif
 
 typedef quote3_error_t (*sgx_ql_get_quote_config_t)(
-    const sgx_ql_pck_cert_id_t *p_pck_cert_id,
-    sgx_ql_config_t **pp_quote_config);
+    const sgx_ql_pck_cert_id_t* p_pck_cert_id,
+    sgx_ql_config_t** pp_quote_config);

```

Comment:
Have you consciously chosen to align this code with the C++ coding conventions (http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#main) or is this a gratutous change throughout the file? If this was made to align with C++ coding conventions, that's cool, but if so, you should probably do that in a separate commit.